sql/src/planner.rs

```diff
@@ -741,13 +815,78 @@ fn parse_column(col: &ColumnDef) -> Result<ColumnSchema> {
     let builder = column_schema::Builder::new(col.name.value.clone(), data_type)
         .is_nullable(is_nullable)
         .is_tag(is_tag)
-        .comment(comment);
+        .comment(comment)
+        .default_value(default_value);
 
     builder.build().context(InvalidColumnSchema {
         column_name: &col.name.value,
     })
 }
 
+// Analyze default value exprs.
+fn analyze_column_default_value_options<'a, P: MetaProvider>(
```

Comment:
I think we can persist casted `DfLogicalExpr`, because `DfLogicalExpr` can be serialized, but `PhysicalExpr` does not implement similar trait.

Reply:
Yes, we can persist the final physical exprs.

To achieve this, we need import `arrow-ballista` which contains serialize api for datafusion's `PhysicalExpr`. Futher more, maybe we need maintain a fork of arrow-ballista which depends on datafusion.

https://github.com/apache/arrow-ballista/blob/e2a7a004fe21c57d632673ae58a7348d7563e625/ballista/rust/core/src/serde/physical_plan/to_proto.rs#L162-L164

Reply:
TIL. I don't find this in `arrow-datafusion` 🙈

This approach is doable but might require another colossal change by either fork `arrow-ballista` or update `arrow-datafusion`. We can add one item in #252 for persisting final physical exprs.

There is also an issue in `arrow-datafusion` https://github.com/apache/arrow-datafusion/issues/3257 that may move that part into DF. Then `Serializeable` can also be implemented to `PhysicalExpr`. I prefer this way if both are ok.

Reply:
New issue(#259) added.